app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -135,16 +135,30 @@ def stop_middleware_server(ems_ref)
       run_generic_operation(:Shutdown, ems_ref)
     end
 
-    def stop_middleware_deployment(ems_ref)
+    def stop_middleware_deployment(ems_ref, name)
```

Comment:
> if this is fixing #9769, please link to it properly in the git commit message:
> 
> It doesn't really fix it but it is the MIQ part of the fix. The rest of the fix is on the provider side. So I'm not sure if it should be closed when this is merged because the issue could still be seen by QA or whomever.

@jshaughn 
so this is not fixed yet on hawkular side?